ports/nrf/boards/pitaya_go/pins.c

```diff
@@ -58,7 +58,7 @@ STATIC const mp_rom_map_elem_t board_module_globals_table[] = {
   { MP_ROM_QSTR(MP_QSTR_LED_GREEN), MP_ROM_PTR(&pin_P0_11) },
   { MP_ROM_QSTR(MP_QSTR_LED_BLUE), MP_ROM_PTR(&pin_P0_12) },
 
-  { MP_ROM_QSTR(MP_QSTR_USR_BTN), MP_ROM_PTR(&pin_P1_00) },
+  { MP_ROM_QSTR(MP_QSTR_BUTTON_USER), MP_ROM_PTR(&pin_P1_00) },
```

Comment:
Only one thing: this is `BUTTON_USER`, and the others are `BUTTON_USR`. Did you mean `USR`? I looked at the silk and I don't see a label, but maybe I missed it.

Reply:
Never mind. I see it says "USER" on the silk in the middle of the board. I missed that first time around.